lti_emailer/canvas_api_client.py

```diff
@@ -21,6 +21,21 @@
 SDK_CONTEXT = SessionInactivityExpirationRC(**settings.CANVAS_SDK_SETTINGS)
 
 
+def get_section(canvas_course_id, section_id):
```

Comment:
In another page, with no guarantee that it'll still be in the cache when this page gets hit.